libc-test/build.rs

```diff
@@ -4476,14 +4476,14 @@ fn test_linux(target: &str) {
         (struct_ == "iw_encode_ext" && field == "key") ||
         // the `tcpi_snd_rcv_wscale` map two bitfield fields stored in a u8
         (struct_ == "tcp_info" && field == "tcpi_snd_rcv_wscale") ||
-        // the `tcpi_delivery_rate_app_limited` field is a bitfield on musl
-        (musl && struct_ == "tcp_info" && field == "tcpi_delivery_rate_app_limited") ||
-        // the `tcpi_fast_open_client_fail` field is a bitfield on musl
-        (musl && struct_ == "tcp_info" && field == "tcpi_fast_open_client_fail") ||
```

Comment:
Any idea why these aren't a problem anymore?

Reply:
These fields from newer version of musl are currently only available on the loongarch64 bindings.